projects/core/src/user/facade/user.service.ts

```diff
@@ -267,6 +260,13 @@ export class UserService {
     return this.store.pipe(select(fromStore.getAddressesLoading));
   }
 
+  /**
+   * Returns an action processing flag
+   */
+  getAddressActionProcessingStatus(): Observable<boolean> {
```

Comment:
Please add a test in `user.service.spec.ts` for this new method

Reply:
Test added.